src/index.html

```diff
@@ -1,13 +1,410 @@
+          <div class="nav__top-nav">
+            <label
+              class="icon icon--cross"
+              for="menu-checkbox"
+            >
+            </label>
+            <a class="logo" href="/">
+              <img
+                class="logo__img"
+                src="./images/logo.png"
+                alt="Miami Condo Kings"
+              >
+            </a>
+          </div>
+
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#about-us" class="nav__link">About Us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#gallery" class="nav__link">gallery</a>
+            </li>
+            <li class="nav__item">
+              <a href="#testimonials" class="nav__link">testimonials</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contact" class="nav__link">Contacts</a>
+            </li>
+          </ul>
+
+          <div class="nav__calls">
+            <a class="nav__call-link" href="tel: +12345555555">+1 234 555-55-55</a>
+            <a class="nav__call-link nav__call-link--border" href="tel: +12345555555">call to order</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main class="page__main">
+      <!-- About Us:  -->
+      <div class="page__section grid grid--desktop">
+        <div class="grid__item grid__item--1-3">
+          <h2 id="about-us" class="about-us__title">
+            Miami best
+            <br>Real estate <a href="https://twitter.com/MiamiCondoKings" class="about-us__title-link">here</a>
+          </h2>
+        </div>
+        <div class="grid__item  grid__item--7-12">
+          <section class="about-us">
+            <div class="about-us__content">
+              <a
+                href="https://www.facebook.com/Miamicondokings/"
+                class="about-us__subtitle"
+              >
+                Over 50 years experience in love.
+              </a>
+
+              <p class="about-us__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+                <br>
+                <br>The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.
+                <br>Working directly with developers - so have more useful information about the profitable deals.
+                <br>
+                <br><strong>We will never be late!</strong>
+                <br><strong>Guaranteed secure transaction - 100%</strong>
+              </p>
+            </div>
+          </section>
+        </div>
+      </div>
```

Comment:
1. In my opinion, this is one big section called “Best Real Estate in Miami” (no section inside).
2. I think this is too much use of `<br>`, it is better to use several paragraphs instead of `<br>` (and semantically better).